client/my-sites/domains/domain-management/security/style.scss

```diff
@@ -0,0 +1,33 @@
+.security {
+	.security__header {
+		display: flex;
+		align-items: center;
+
+		span:first-child {
+			margin-right: auto;
+		}
+		.security__status {
+			display: flex;
+			align-items: center;
+			border-bottom: 0;
+			padding-bottom: 0;
+			font-weight: 600;
+			svg {
+				margin-right: 5px;
+			}
+
+			&.disabled svg {
+				fill: var( --studio-pink-50 );
```

Comment:
Can we change this to `--color-error` so that it'll carry over to the different Calypso themes appropriately?